tools/pipelines/templates/include-policy-check.yml

```diff
@@ -61,7 +61,7 @@ stages:
         inputs:
           targetType: 'inline'
           script: |
-            git status | grep -v -E 'package.json|package-lock.json|packageVersion.ts|lerna.json|.npmrc|build-tools/.npmrc|\(use.*' | grep '^\s' > git_status.log
+            git status | grep -v -E 'package.json|package-lock.json|packageVersion.ts|lerna.json|.npmrc|build-tools/.npmrc|*packages/tools/devtools/devtools-view/__screenshots__/*|\(use.*' | grep '^\s' > git_status.log
```

Comment:
I'm a bit hesitant about this one (and the similar change in `build-npm-package.yml`... I think in CI we shouldn't expect these files to be changed, so if they do, it's better if the policy fails. @Josmithr thoughts?

Reply:
Correct. We don't expect these to be different. If they are, that means the tests failed.

Reply:
Then should I remove this and add expect the empty files check in `fluidBuild.config.cjs`?

Reply:
I don't think we want to add anything to either place. When CI runs these tests, it shouldn't generate any changes. If it generates changes, that means the tests failed and generated new baseline images.

Reply:
I see. So if there is new image generated means the UI got modified and we should let this step fail?

Reply:
Exactly, yeah. Either that or something about our Git LFS configuration has gone wrong and our tooling thinks there are differences when there aren't, which is a case we want to be aware of since it would disrupt developer workflows too.